p4-16/spec/P4-16-spec.mdk

```diff
@@ -1496,12 +1495,21 @@ Each parameter may be labeled with a direction:
   call, the value of the parameter is copied to the corresponding storage location
   for that l-value.
 - `inout` parameters are both `in` and `out`. An argument passed as
-    an `inout` parameter must be an l-value.
-- No direction indicates that value of parameter is either:
-	- a compile-time known value
-	- an action parameter that can only be set by the control plane
-	- an action parameter that can be set directly by another calling
-      action; in this case it behaves like an `in` parameter
+  an `inout` parameter must be an l-value.
+- The meaning of parameters with no direction depends upon the kind of
+  entity the parameter is for:
+  - For anything other than an action, e.g. a control, parser, or
+    function, a directionless parameter means that the value supplied
+    as an argument in a call must be a compile-time known value.
+  - For an action, a directionless parameter indicates:
+    - If the action appears in an `actions` table property (Section
+      [#sec-table-action-list]), then the parameter's value must be
+      provided by the control plane software when an entry is added to
+      the table that uses that action.
```

Comment:
@jafingerhut The phrase "must be provided by the control plane software ..." should be edited, perhaps to refer to some of the new text in this PR: https://github.com/p4lang/p4-spec/issues/914

Reply:
Updated the text to make it more consistent with #914.